packages/rancher-pushprox/charts/templates/pushprox-clients-rbac.yaml

```diff
@@ -11,7 +11,12 @@ rules:
   resourceNames:
   - {{ template "pushProxy.client.name" . }}
 {{- if and .Values.clients.https.enabled .Values.clients.https.useServiceAccountCredentials }}
-- nonResourceURLs: ["/metrics"]
+- nonResourceURLs:
+  - "/metrics"
+  - "/metrics/cadvisor"
```

Comment:
`/metrics/cadvisor` should not be hard-coded; instead, it should come from the `path` provided under all `additionalEndpoints`.